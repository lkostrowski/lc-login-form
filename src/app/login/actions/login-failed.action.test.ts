import { createLoginFailedAction } from './login-failed.action';

describe('Login Failed Action', function() {
    it('Should match snapshot', () => {
        expect(
            createLoginFailedAction({
                error: {
                    message: 'test error',
                },
            }),
        ).toMatchSnapshot();
    });
});
