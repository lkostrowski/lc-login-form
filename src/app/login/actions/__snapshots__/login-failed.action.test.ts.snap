// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Failed Action Should match snapshot 1`] = `
Object {
  "payload": Object {
    "error": Object {
      "message": "test error",
    },
  },
  "type": "login:login-failed",
}
`;
